Composer/packages/client/src/components/ProjectTree/treeItem.tsx

```diff
@@ -113,58 +111,81 @@ const navItem = (isActive: boolean, isSubItemActive: boolean) => css`
 export const overflowSet = css`
   width: 100%;
   height: 100%;
-  padding-left: 12px;
   padding-right: 12px;
   box-sizing: border-box;
-  line-height: 36px;
+  line-height: 24px;
   justify-content: space-between;
   display: flex;
-  justify-content: space-between;
 `;
 
+const statusIcon = {
+  width: '24px',
+  height: '18px',
+  fontSize: 16,
+  marginLeft: 6,
+};
+
 const warningIcon = {
-  marginRight: 5,
+  ...statusIcon,
   color: '#BE880A',
-  fontSize: 9,
+};
+
+const errorIcon = {
+  ...statusIcon,
+  color: '#CC3F3F',
 };
 
 // -------------------- TreeItem -------------------- //
 
 interface ITreeItemProps {
-  link: any;
-  isActive: boolean;
+  link: TreeLink;
+  isActive?: boolean;
   isSubItemActive?: boolean;
-  depth: number | undefined;
-  onDelete: (id: string) => void;
-  onSelect: (id: string) => void;
+  menu?: TreeMenuItem[];
+  onSelect?: (link: TreeLink) => void;
+  icon?: string;
+  dialogName?: string;
+  showProps?: boolean;
+  shiftOut?: number; // needed to make an outline look right; should be the size of the "details" reveal arrow
```

Comment:
wht does shiftOut mean? Better name to this prop?

Reply:
Yeah, I admit it's not a great name. It should be something like "forceIndent".